addon/webextension/background.js

```diff
@@ -77,10 +77,20 @@ class PersistentPageModificationEffect {
         display: inline-block;
       }
 
+      .donotdelete-revert {
+        transform: ScaleY(1);
+        transition: 1s ease-in;
+      }
+
+      /* after revert, revert the tooltip */
+      .donotdelete-revert .donotdelete-tooltip {
+        transform: scaleY(1) translateY(-50%);
+      }
+
       .donotdelete-tooltip {
```

Comment:
Can you also add `font-style: normal !important;` under "Neutralizing styles" for this selector? Just saw the text italicized otherwise on New York Times' website.